### ISSUE TITLE
Please add legacy bindings to the key-map

### ISSUE THREAD

**Issue by sten0** [rnkn/olivetti#35]
Hi @rnkn,

I think I understand why you made the change from `C-c [` to `C-c {` (eg: consistency with upstream bindings in f72920f8021196e1518bf32e59ccae09b5948f7e), but I find them horrible on an ANSI keyboard.  <ctrl> + c, then <shift> + [  ...slow and awkward.  In Debian we also like to avoid disrupting the workflow of existing users, if at all possible, so I was originally thinking about reverting this change in our package.

A new team member helpfully suggested checking if anything else uses the C-c [ keybinding among the modes users are likely to combine with Olivetti-mode, and it seems none do, thus I plan to enable legacy bindings in the Emacs package.  Would you also like to provide them to your MELPA and run-from-git-repo users?  If so, I'll file a PR rather than making a Debian-specific change :-)

Best,
Nicholas

**Comment by rnkn** [comment]
I received complaints that the original bindings conflicted with Org Mode for `org-agenda-file-to-front` and `org-remove-file`. I think the consistency with Emacs’ builtin bindings, even if awkward for the fingers, is worthwhile. I understand as the code stands the bindings are a bit hardcoded, so if you’re interested, a good PR would implement the following style method of customisation:
~~~elisp
(defun other-window-and-beyond (count &optional all-frames)
  "Select another window in cyclic ordering of windows.
Successive pressing `o' calls `other-window'."
  (interactive "p")
  (let ((key-vector (where-is-internal this-command
                                       overriding-local-map t)))
    (set-transient-map
     (let ((map (make-sparse-keymap)))
       (define-key map (vector (aref key-vector (1- (length key-vector))))
         'other-window)
       map)
     t)
    (other-window count all-frames)))
~~~
I’m not 100% sure but I think this would allow both the spare keymap and user key customisation.

**Comment by rnkn** [comment]
p.s. If you feel like tackling this, feel free to use the seq library and bump the Emacs version requirement to 25.

**Comment by rnkn** [comment]
Resolved in c895319a66ad545341c6ab5fdb6205c85d79bb77
